osu.Game.Rulesets.Taiko/Skinning/Argon/ArgonCirclePiece.cs

```diff
@@ -7,38 +7,45 @@
 using osu.Framework.Graphics.Colour;
 using osu.Framework.Graphics.Shapes;
 using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics;
 using osu.Game.Graphics.Containers;
 using osu.Game.Rulesets.Objects.Drawables;
 using osu.Game.Rulesets.Taiko.Objects;
 using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Taiko.Skinning.Argon
 {
-    public abstract partial class ArgonCirclePiece : BeatSyncedContainer
+    public abstract partial class ArgonCirclePiece : BeatSyncedContainer, IHasAccentColour
     {
         public const float ICON_SIZE = 20 / 70f;
 
         private const double pre_beat_transition_time = 80;
 
         private const float kiai_flash_opacity = 0.15f;
 
-        private ColourInfo accentColour;
+        private ColourInfo argonAccentColour;
 
         /// <summary>
         /// The colour of the inner circle and outer glows.
         /// </summary>
-        public ColourInfo AccentColour
+        public ColourInfo ArgonAccentColour
```

Comment:
This cannot exist.

Reply:
My idea is to change DrawableHitObject's AccentColour and IHasAccentColour to ColourInfo so that it can support multiple colors.
However, this may affect a lot of things. I currently use the smallest possible cost to temporarily solve this problem.